OrbitBase/include/OrbitBase/Profiling.h

```diff
@@ -9,7 +9,9 @@
 #include <Windows.h>
 using pid_t = uint32_t;
 #else
+#include <stdint.h>
 #include <sys/syscall.h>
+#include <time.h>
```

Comment:
Is this intended?

Reply:
Yes, removing <string> and GlUtils.h from Timer.h uncovered those two missing includes.